deploy-manage/monitor/kibana-task-manager-health-monitoring.md

```diff
@@ -59,13 +55,17 @@ xpack.task_manager.monitored_task_execution_thresholds:
 
 ## Consuming health stats [task-manager-consuming-health-stats]
 
-The health API is best consumed by via the `/api/task_manager/_health` endpoint.
+The health API is best consumed using the `/api/task_manager/_health` endpoint.
 
 Additionally, there are two ways to consume these metrics:
 
-**Debug logging**
+### Debug logging
+```{applies_to}
+deployment:
+  self:
+```
```

Comment:
I think this could probably apply to ECK and ECE too, but it's not a big deal to leave it like this.
It's similar to your comment in the PR description.